requirements/edx/base.txt

```diff
@@ -271,4 +271,4 @@ django-mptt-admin==0.4.2
 -e lms/lib/ucdc_edx_api_plugin
 
 # UCDC SSO and API integration
-git+https://github.com/raccoongang/edx-oauth-client.git@ucdc-v.0.1.1#egg=edx_oauth_client==ucdc-v.0.1.1
+git+https://github.com/raccoongang/edx-oauth-client.git@ucdc-v.0.1.2#egg=edx_oauth_client==ucdc-v.0.1.2
```

Comment:
Have you checked that this requirement will override the previously installed one? I believe it won't. `-e` flag for editing mode could improve this issue.

Reply:
You're right. Interesting how it worked before.
Done